AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -48,6 +48,12 @@
 
 - (instancetype)init;
 
+/** Specifies how children are aligned horizontally. The stack direction will become immutable after this call. */
+- (void)setHorizontalAlignment:(ASHorizontalAlignment)alignment;
```

Comment:
Sorry but I don't get what you meant by 3). Could you please elaborate it?

Btw, other comments were addressed.